src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -25,151 +26,197 @@ export const VET360_TRANSACTION_UPDATE_FAILED =
 export const VET360_TRANSACTION_CLEARED = 'VET360_TRANSACTION_CLEARED';
 export const VET360_CLEAR_TRANSACTION_STATUS =
   'VET360_CLEAR_TRANSACTION_STATUS';
-
-export function clearTransactionStatus() {
-  return {
-    type: VET360_CLEAR_TRANSACTION_STATUS,
-  };
-}
-
-export function fetchTransactions() {
-  return async dispatch => {
-    try {
-      let response;
-      if (isVet360Configured()) {
-        response = await apiRequest('/profile/status/');
-      } else {
-        response = { data: [] };
-        // Uncomment the line below to simulate transactions being processed during initialization
-        // response = localVet360.getUserTransactions();
-      }
-      dispatch({
-        type: VET360_TRANSACTIONS_FETCH_SUCCESS,
-        data: response.data,
-      });
-    } catch (err) {
-      // If we sync transactions in the background and fail, is it worth telling the user?
+export const ADDRESS_VALIDATION_CONFIRM = 'ADDRESS_VALIDATION_CONFIRM';
+export const ADDRESS_VALIDATION_ERROR = 'ADDRESS_VALIDATION_ERROR';
+
+export const clearTransactionStatus = () => ({
```

Comment:
would suggest reverting all these conversions of function declarations to function expressions for the sake of reducing noise in this PR.

Reply:
> would suggest reverting all these conversions of function declarations to function expressions for the sake of reducing noise in this PR.

@erikphansen please clarify, do you want @bradl3yC to change the ES6 arrow functions with es5 function declarations ?

Reply:
Yes, not because it's bad or wrong, but because it makes it harder to tell what the _real_ changes to this file are. The attached screenshot makes it look like a ton of code changed when really it was almost all a change in the indentation level. If we want change existing function declarations to function expressions, we can do that as a separate PR with an explanation of the reasoning.

![image](https://user-images.githubusercontent.com/20728956/69090808-f3360100-09fd-11ea-8e43-ff09b5efd191.png)